packages/nimble-components/src/table/types.ts

```diff
@@ -95,8 +95,8 @@ export interface TableRecordHierarchyOptions {
 
 export const TableRecordDelayedHierarchyState = {
     none: undefined,
-    canLoadChildren: 'canLoadChildren',
-    loadingChildren: 'loadingChildren'
+    canLoadChildren: 'can-load-children',
```

Comment:
Looks like TableRowSelectionState has the same inconsistency and appears to be all internal, Could we clean that one up as well?

Reply:
Yes, I can update that. It's an internal enum, so I wouldn't expect it to ever be used in or cause problems in Blazor. However, it seems like a good idea to go ahead and make it align with out conventions.